internal/controller/server_controller_test.go

```diff
@@ -276,9 +276,15 @@ var _ = Describe("Server Controller", func() {
 		probeAgent := probe.NewAgent(server.Spec.UUID, registryURL, 100*time.Millisecond)
 		go func() {
 			defer GinkgoRecover()
+			time.Sleep(550 * time.Millisecond) // force discovery timeout
```

Comment:
Can we make this a little bit shorter?

Reply:
it will become unpredictable, since it depends on the hardware how quick your reconcile loops are

Reply:
I reduced it, lets see if it will fail on other hardware ;)

Reply:
might want to use a constant instead of hardcode ?